lib/dpdk/CMakeLists.txt

```diff
@@ -4,17 +4,19 @@ find_package(NUMA REQUIRED)
 
 add_library(kotekan_dpdk dpdkCore.cpp zeroSamples.cpp invalidateVDIFframes.cpp)
 
-if(DPDK_FOUND)
-    # The new way to include DPDK with pkg-config
+# Select integration path based on effective mode from FeatureDPDK.cmake
+if(DEFINED KOTEKAN_DPDK_MODE AND (KOTEKAN_DPDK_MODE STREQUAL "NEW"))
+    # NEW: use pkg-config provided flags from FeatureDPDK. If missing (rare), require them here.
+    if(NOT DPDK_FOUND)
+        find_package(PkgConfig REQUIRED)
+        pkg_check_modules(DPDK REQUIRED libdpdk>=19.11)
+    endif()
     add_definitions(${DPDK_STATIC_CFLAGS})
     set(DPDK_LIBRARIES ${DPDK_STATIC_LDFLAGS})
     message(STATUS "DPDK include dirs: ${DPDK_INCLUDE_DIRS}")
     target_include_directories(kotekan_dpdk SYSTEM PRIVATE ${DPDK_INCLUDE_DIRS})
 else()
-    # Link in DPDK with the find script
-    find_package(DPDK REQUIRED)
-    message("DPDK include dir: ${DPDK_INCLUDE_DIR}")
-    target_include_directories(kotekan_dpdk SYSTEM PRIVATE ${DPDK_INCLUDE_DIR})
+    message(FATAL_ERROR "kotekan_dpdk requested but KOTEKAN_DPDK_MODE is not NEW")
```

Comment:
Do we actually need that `NEW` distinction anymore?  It's really just version > 19.11